wger/manager/views/day.py

```diff
@@ -96,7 +106,7 @@ class DayCreateView(DayView, CreateView):
     Generic view to add a new exercise day
     '''
 
-    title = ugettext_lazy('Add workout day')
+    # title = ugettext_lazy('Add workout day')
```

Comment:
I would rather you remove this line of commented code, it will make the code look neat.

Reply:
thank you for that i have removed it